mllib/src/main/scala/org/apache/spark/mllib/tree/model/InformationGainStats.scala

```diff
@@ -38,11 +39,17 @@ class InformationGainStats(
     val leftPredict: Predict,
     val rightPredict: Predict) extends Serializable {
 
+  /**
+   * @since 1.0.0
+   */
   override def toString: String = {
     s"gain = $gain, impurity = $impurity, left impurity = $leftImpurity, " +
       s"right impurity = $rightImpurity"
   }
 
+  /**
+   * @since 1.2.0
+   */
```

Comment:
We need not have tags for hashCode, equals and toString. Please get rid of them here and everywhere.